corehq/form_processor/management/commands/run_submission_reprocessing_queue.py

```diff
@@ -35,8 +35,12 @@ def get_fetching_interval(self):
     def get_items_to_be_processed(cls, utcnow):
         _record_datadog_metrics()
         queued_threshold = utcnow - timedelta(minutes=ENQUEUING_TIMEOUT)
-        queue_filter = Q(date_queued__isnull=True) | Q(date_queued__lte=queued_threshold)
-        query = UnfinishedSubmissionStub.objects.filter(queue_filter).order_by('timestamp')
+        min_processing_age = datetime.utcnow() - timedelta(minutes=5)
+        filters = Q(date_queued__isnull=True) | Q(date_queued__lte=queued_threshold)
+
+        # wait 5 mins before processing to avoid processing during form submission
```

Comment:
This seems a bit brittle. What happens if it it's processed during form submission?

Reply:
It doesn't really matter, it's just duplicate work.